src/patternfly/components/Button/button.scss

```diff
@@ -263,7 +263,6 @@
 
   // Count
   --#{$button}__count--MarginInlineStart: var(--pf-t--global--spacer--sm);
-  --#{$button}--disabled__c-badge--BorderColor: var(--pf-t--global--border--color--disabled);
   --#{$button}--disabled__c-badge--BackgroundColor: var(--pf-t--global--background--color--disabled--default);
```

Comment:
This should also move the the badge styles for `.pf-m-disabled` so it's available when we use the disabled badge in other disabled things (like menu toggle). We should also probably have a disabled badge example and document the `.pf-m-disabled` class in the docs table - that would also be a good place to mention something like "Applies disabled badge styling. For use when a badge is used in a disabled element."